### DISCUSSION TITLE
Perfect Mode?

### DISCUSSION THREAD

**Discussion by AbnormalPoof** [FunkinCrew/Funkin#1853] [Q&A] [answered]
I saw in PlayState a few references to a thing called "Perfect Mode"
Attempting to enable it crashes the game

What is this "Perfect Mode" and what was its purpose?

**Comment by Tr1NgleDev** [comment]
i think its maybe was debug thing

**Reply by Windows1Door2Daniel** [reply]
Most likely

**Comment by ninjamuffin99** [comment, marked as answer]
was a debug thing, essentially a "bot play" or whatever. was gonna be for when i record videos for twitter, but I think i prefer recording the videos myself of me just playing the level hehehe so i never bothered to finish it or go any further

**Reply by AbnormalPoof** [reply]
interesting
